app/views/athletes/index.html.erb

```diff
@@ -1 +1,9 @@
+<h1>Athletes</h1>
+
+<% Athlete.all.each do |athlete| %>
```

Comment:
Try to make this call in the controller instead and make use of instance variables in the view. 
```
def index
   @athletes = Athlete.all 
end 
```
and in the view. 
``` 
<% @athletes.each do |athlete| %>

```

That is better practice :)